src/main/java/net/mcreator/element/types/Biome.java

```diff
@@ -154,6 +155,43 @@ public boolean hasVines() {
 		return !treeVines.isEmpty();
 	}
 
+	public boolean hasStructure(String structureType) {
+		return switch (structureType.toLowerCase(Locale.ENGLISH)) {
+			case "mineshaft": yield spawnMineshaft;

```

Comment:
This can as well be 'expression type' switch, _like_ this:
```suggestion
			case "mineshaft" -> spawnMineshaft;
```